src/main/java/pl/edu/pwr/kpz/lostanimalsbackend/model/dto/LostReportRequestDTO.java

```diff
@@ -1,16 +1,26 @@
 package pl.edu.pwr.kpz.lostanimalsbackend.model.dto;
 
-import lombok.*;
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Past;
+import lombok.Data;
+import org.springframework.format.annotation.DateTimeFormat;
 import pl.edu.pwr.kpz.lostanimalsbackend.model.entities.Coordinate;
 
 import java.util.Date;
 
 @Data
 public class LostReportRequestDTO {
+    @Past(message = "The date should be in the past ")
+    @DateTimeFormat(pattern="dd-MM-yyyy")
     private Date lostDate;
+    @NotNull(message = "Lost report coordinate shouldn't be null")
     private Coordinate coordinate;
+    @NotNull(message = "Lost report description shouldn't be null")
```

Comment:
To akurat, przynajmniej według bazy, może być nullowane

Reply:
git to trzeba jeszcze przedyskutować i na pewno dodać że email i numer telefonu ma być unique